.circleci/config.yml

```diff
@@ -769,6 +794,7 @@ workflows:
             - lint
 
       - test:
+          context: "raiden-general-2021"
```

Comment:
why do we need the context here?

Reply:
The benchmark `conftest.py` gets imported which in turn imports `_codespeed.py` which needs the environment variables to operate.

Reply:
hmmm. I don't like it too much. the whole idea of contexts is to avoid spilling secrets everywhere.
where exactly do the environment variables come into play?